build/ansible/pmm-docker/update.yml

```diff
@@ -170,3 +170,8 @@
       file:
         state: absent
         path: /usr/share/pmm-server/maintenance/maintenance.html
+
+    - name: Remove the file provisioned by 'getStatus'
+      file:
+        state: absent
+        path: /srv/pmm-update.json
```

Comment:
I don't think that we call it after upgrade. seems like we need to cleanup this file from leftovers of old UI upgrade process.

Reply:
I think we are still running this playbook when building PMM - https://github.com/percona/pmm/blob/v3/build/docker/server/Dockerfile.el9#L32

However, as you pointed out, we may need to get rid of it in view of the new update workflow implemented in v3.

Reply:
I mean that we still have file creation/removals, readiness checks that was required for UI update, but not a case anymore

Reply:
Removed.